### ISSUE TITLE
NullPointerException in initiateToolTip()

### ISSUE THREAD

**Issue by benfry** [processing/processing#3286]
```
Exception in thread "AWT-EventQueue-0" java.lang.NullPointerException
    at javax.swing.ToolTipManager.initiateToolTip(ToolTipManager.java:478)
    at javax.swing.ToolTipManager.mouseEntered(ToolTipManager.java:435)
    at processing.mode.java.pdex.JavaTextArea$MouseHandler.mouseEntered(JavaTextArea.java:749)
    at java.awt.AWTEventMulticaster.mouseEntered(AWTEventMulticaster.java:300)
    at java.awt.Component.processMouseEvent(Component.java:6534)
    at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
```

**Comment by benfry** [comment]
I may have broken this one when removing some extraneous listener code, but it looks like one of the mouse listeners in the list is null. Not sure how that's possible.

**Comment by Manindra29** [comment]
Is there a consistent way to reproduce this?

**Comment by benfry** [comment]
This was reported by @shiffman over here: https://github.com/processing/processing/issues/3265

**Comment by shiffman** [comment]
I'm not able to make this error happen anymore now that the debugger issues were resolved.  Shall I close?

**Comment by Manindra29** [comment]
This kind of error was reported before in #2926. But I believe it was taken care of. I can't think of why the Debugger issue would be related to the tooltip one. We can close this one and reopen #2926. It would be nice if there was a way to reproduce the error.

**Comment by benfry** [comment]
I may have bumped something that fixed it. We can close for now and reopen (one of them) if it reappears.

**Comment by shiffman** [comment]
I'm seeing the tooltip error again.  I haven't pinpointed it precisely, but I believe it's appearing during error checking (I was changing things quickly in code and lots of errors were appearing/disappearing.)  Should I re-open?

**Comment by Manindra29** [comment]
This is unlikely to have anything to do with the error checking. This is being caused due to a bug in the tooltip handling for variable/methods tooltips.

**Comment by benfry** [comment]
@Manindra29 are you working on this one?

**Comment by Manindra29** [comment]
No, I'm not at the moment.

**Comment by benfry** [comment]
Let's see if this is fixed by #4113 in 3.0.2.

**Comment by benfry** [comment]
Haven't seen this since.
